qdrant-landing/content/documentation/concepts/storage.md

```diff
@@ -153,16 +159,12 @@ client.CreateCollection(context.Background(), &qdrant.CreateCollection{
 })
 ```
 
-This will create a collection with all vectors immediately stored in memmap storage.
-This is the recommended way, in case your Qdrant instance operates with fast disks and you are working with large collections.
-
-
-- Set up `memmap_threshold` option. This option will set the threshold after which the segment will be converted to memmap storage.
+When you enable **On-Disk** storage, all vectors in a collection are stored on disk immediately. This is recommended if your Qdrant instance uses fast disks and handles large collections.
 
-There are two ways to do this:
+To configure when segments switch to **On-Disk** storage, use the `memmap_threshold` option. You can set this threshold in two ways:
 
-1. You can set the threshold globally in the [configuration file](/documentation/guides/configuration/). The parameter is called `memmap_threshold` (previously `memmap_threshold_kb`).
-2. You can set the threshold for each collection separately during [creation](/documentation/concepts/collections/#create-collection) or [update](/documentation/concepts/collections/#update-collection-parameters).
+1. **Globally**: Adjust the `memmap_threshold` parameter in the [configuration file](/documentation/guides/configuration/).
+2. **Per Collection**: Set the threshold during the [creation](/documentation/concepts/collections/#create-collection) or [update](/documentation/concepts/collections/#update-collection-parameters) of each collection.
```

Comment:
Let's drop this, since the plan is to deprecate this parameter anyway.

Instead we can say to explicitly set `on_disk: true` for:
- vectors (vector storage)
- hnsw
- payload (`on_disk_payload: true`)
- payload indices